test/test_tensorclass_nofuture.py

```diff
@@ -11,7 +11,15 @@
 
 import pytest
 import torch
-import torchsnapshot
+
+try:
+    import torchsnapshot
+
+    _has_torchsnapshot = True
+except ImportError as err:
+    _has_torchsnapshot = False
+    TORCHSNAPSHOT_ERR = str(err)
```

Comment:
this should be created out of the try/except to be visible
TORCHSNAPSHOT_ERR=None will suffice

Reply:
done in the try...except block like we did for TORCHREC